pkg/amqp-ext/Flags.php

```diff
@@ -2,11 +2,11 @@
 
 namespace Enqueue\AmqpExt;
 
-use Interop\Amqp\AmqpConsumer;
-use Interop\Amqp\AmqpDestination;
-use Interop\Amqp\AmqpMessage;
-use Interop\Amqp\AmqpQueue;
-use Interop\Amqp\AmqpTopic;
+use Interop\Amqp\AmqpConsumer as InteropAmqpConsumer;
+use Interop\Amqp\AmqpDestination as InteropAmqpDestination;
+use Interop\Amqp\AmqpMessage as InteropAmqpMessage;
+use Interop\Amqp\AmqpQueue as InteropAmqpQueue;
+use Interop\Amqp\AmqpTopic as InteropAmqpTopic;
```

Comment:
Shouldn't we get rid of these imports and use `Enqueue\AmqpExt\AmqpConsumer` instead.

Reply:
You could, but the same I did above has been done in other classes.
I suggest to keep it that way for now and refactor mid-term. What do you think?